### ISSUE TITLE
Nicehash excavator server wont start

### ISSUE THREAD

**Issue by FunDeestruction** [nicehash/excavator#179]
I'm getting an error that the excavator.exe is not a valid win32 application.   I've tried reinstalling NHm2, again using an older version, I've tried removing old drivers, and installing the most current.  I cant figure out what could be going on.  Just started after the new update.  

I'm running an older gigabyte mobo with an amd processor.   A8 something.  4gb of ram, a 1070ti and a 1080.  Any suggestions are greatly appreciated!

**Comment by dropky** [comment]
Are you running 32-bit version of Windows (we only support 64-bit)? There is also possible that your file got corrupted during download, if you download excavator manually from [github](https://github.com/nicehash/excavator/releases/tag/v1.4.3a), unzip it and try to run it (double click on excavator.exe), does it work? If yes, you can manually copy the content of the zip file to the `C:\Users\username\AppData\Roaming\nhm2\bin\excavator_server` directory.

**Comment by FunDeestruction** [comment]
Thank you!  I'm using win 7 pro 64bit.  I did that and it worked.  I was able to replicate my problem too.  Either Windows firewall or my antivirus was deleting files.  I attempted to restore and ignore but it still didn't work.  The file path must have changed or something with the update.  It didn't seem like that was the issue but apparently it was.  Once I disabled all protection and manually copied those files it worked. 
 Thanks for the help!
